src/main/java/org/cbio/gdcpipeline/GDCPipelineApplication.java

```diff
@@ -33,7 +31,8 @@ private static Options getOptions(String []input){
 		options.addRequiredOption("m","manifest_file",true,"Manifest file path");
         options.addOption("f", "filter_normal_sample", true, "True or False. Flag to filter normal samples. Default is True ");
 		options.addOption("d", "datatypes", true, "Datatypes to run. Default is All");
-		options.addOption("h", "help", true, "shows this help document and quits.");
+        options.addOption("separate_maf", "separate_maf", true, "True or False. Process and generate MAF files individually apart from merging. Default is False");
```

Comment:
picky, but could this be "separate_mafs" plural?

Reply:
done